### PR TITLE
Initial Delivery of Enrich Tests

### PR DESCRIPTION
**Pull request by rguderian**

### CODE REVIEW

File: clinical-enrichment/test/src/main/resources/scenarios/phi_FHIR_Patient_DELETE.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
```
Comment (ibmatc) [inline clinical-enrichment/test/src/main/resources/scenarios/phi_FHIR_Patient_DELETE.json:1]: might be dangerous to label this "phi*" - raises red flags about what's being stored. It's not truly PHI in this environment, just input data that's been normalized/enhanced, but not deid'ed

File: clinical-enrichment/test/src/main/java/tests/Enrich_Flow.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+import categories.BasicEnrichment;
+
+
+@TargetEnv("enrich-flow.properties")
+@RunWith(ZeroCodeUnitRunner.class)
+
+public class Enrich_Flow {
+	
+//	 @Ignore
```
Comment (ibmatc) [inline clinical-enrichment/test/src/main/java/tests/Enrich_Flow.java:29]: should //@Ignore lines be removed at this point?

File: clinical-enrichment/test/src/main/java/utilities/PhiDeIDCompare.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    		@JsonProperty("dataName")String dataName,
+            @JsonProperty("phi")String phiString,
+            @JsonProperty("deid")String deidString) {
+        this.dataName = dataName;
+    	this.phiString = phiString;
+        this.deidString = deidString;
+        this.result = true;
+    }
+
+    public String getdeidString() {
```
Comment (ibmatc) [inline clinical-enrichment/test/src/main/java/utilities/PhiDeIDCompare.java:35]: Fix camelCase naming: "getDeidString()"

**Review by ibmatc — APPROVED** [review]
LGTM

File: clinical-enrichment/test/src/main/java/utilities/ZerocodeUtilities.java [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+        
+    }
+    
+    public PhiDeIDCompare notEqual(PhiDeIDCompare pd) throws InterruptedException {
+        
+    	System.out.println("input object: "+pd.toString());
+   	
+    	String phiStr = pd.getphiString();
+    	String deidStr = pd.getdeidString();
+
```
Comment (ranum) [inline clinical-enrichment/test/src/main/java/utilities/ZerocodeUtilities.java:58]: Do you need null checks here?

File: clinical-enrichment/test/src/main/java/utilities/ZerocodeUtilities.java [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    }
+    
+    public PhiDeIDCompare notEqual(PhiDeIDCompare pd) throws InterruptedException {
+        
+    	System.out.println("input object: "+pd.toString());
+   	
+    	String phiStr = pd.getphiString();
+    	String deidStr = pd.getdeidString();
+    	
+    	if(phiStr.equals(deidStr)) {
```
Comment (ibmatc) [inline clinical-enrichment/test/src/main/java/utilities/ZerocodeUtilities.java:59]: do we need to check if phiStr!=null or deidStr!=null here?

File: clinical-enrichment/test/src/main/resources/scenarios/enrich_DEID_Prep_No_Store_Patient.json [hunk 6]
```diff
@@ -0,0 +1,112 @@
+{
+    "scenarioName": "********enrich_DEID_Store_Patient********",
```
Comment (ranum) [inline clinical-enrichment/test/src/main/resources/scenarios/enrich_DEID_Prep_No_Store_Patient.json:2]: no store

**Pull request merged**